src/scry/completion/dependency_graph.cr

```diff
@@ -122,16 +122,17 @@ module Scry::Completion::DependencyGraph
 
     def parse_requires(file_path)
       file_dir = File.dirname(file_path)
-      File.each_line(file_path).reduce([] of String) do |acc, line|
+      paths = [] of String
+      File.each_line(file_path) do |line|
         require_statement = /^\s*require\s*\"(?<file>.*)\"\s*$/.match(line)
         unless require_statement.nil?
```

Comment:
Note: can be written as `if require_statement`
But that's not related to this pr